mcnode/api.go

```diff
@@ -328,6 +327,81 @@ func (node *Node) httpDelete(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, count)
 }
 
+// datastore interface
+type DataObject struct {
+	Data []byte `json:"data"`
```

Comment:
How do these tags work? `Data` is decoded from `data` field in json, i.e. it's `ABC` in `{ "data": "ABC" }`?